src/script/components/MessagesList/MessageList.tsx

```diff
@@ -245,78 +255,167 @@ export const MessagesList: FC<MessagesListParams> = ({
     syncScrollPosition();
   };
 
+  const jumpToLastMessage = () => {
+    if (conversation) {
+      // clean up anything like search result
+      setHighlightedMessage(undefined);
```

Comment:
It feels that this cleanup should happen in the `useEffect` that listens to `initialMessage` change, in block `if(!initialMessage)`, but I'm afraid to break something this way. I removed the `conversation.initialMessage(undefined)` from here, as it will trigger the messages reload. In case of `amplify.publish(WebAppEvents.CONVERSATION.SHOW, conversation, {});` it should be cleaned anyways, and if last messages are loaded, it should be cleaned only if the search result was shown